pkg/service/record/record.go

```diff
@@ -19,38 +19,37 @@ import (
 var Emoji = "\U0001F430" + " Keploy:"
 
 type recorder struct {
-	logger *zap.Logger
+	Logger *zap.Logger
```

Comment:
Why do we need to export logger, since it is called internally by the exported functions.

Reply:
reverted.